laravel/resources/views/schedules/resident/milestone.blade.php

```diff
@@ -2,133 +2,59 @@
-
-                <select name="milestones2" id="milestones2" required>
-					<option value="" selected> -- Select a Milestone -- </option>
-					@if(!is_null($milestones))
-						@foreach ($milestones as $milestone)
-							<?php
-								$milestone_id=$milestone['id'];
-								$category = $milestone['category'];
-								$title = $milestone['title'];
-								$detail = $milestone['detail'];
-								echo '<option value="'.$milestone_id.'" title="'.$title.'">'.$category.' - '.$detail.'</option>';
-							?>
-						@endforeach
-					@endif
-                </select>
-
-                <br>
-
-                <label>What is your educational objective for this OR today?</label><br>
-
-                <textarea rows="3" name="objectives2" id="objectives2" class="form-control" required></textarea>
-				
-				<div id="anesthesiologist_preference">
-					<label>Anesthesiologist Preference:</label>
-					<br>
-					<select name="pref_anest2" id="pref_anest2" class="PreferenceSelector">
-						<option selected="selected">No Preference</option>
-						@foreach($anesthesiologists as $a)
-							<option value="{{ $a->id }}">Dr. {{ $a->first_name }} {{ $a->last_name }}</option>
-						@endforeach
-					</select>
-				</div>
-				<br>
-	        @endif
-
-
-	        @if(is_null($data3['schedule']))
-				<h5>Your 3rd Preference: None</h5>
-			@else
-				<h5>Your 3rd Preference: Room {{ $data3['schedule']['room'] }} with {{ $data3['attending'] }}</h5>
-                <label>Select your Milestone:</label><br>
-
-                <select name="milestones3" id="milestones3" required>
-					<option value="" selected> -- Select a Milestone -- </option>
-					@if(!is_null($milestones))
-						@foreach ($milestones as $milestone)
-							<?php
-								$milestone_id=$milestone['id'];
-								$category = $milestone['category'];
-								$title = $milestone['title'];
-								$detail = $milestone['detail'];
-								echo '<option value="'.$milestone_id.'" title="'.$title.'">'.$category.' - '.$detail.'</option>';
-							?>
-						@endforeach
-					@endif
-                </select>
-
-                <br>
-
-                <label>What is your educational objective for this OR today?</label><br>
-
-                <textarea rows="3" name="objectives3" id="objectives3" class="form-control" required></textarea>
-				
-				<div id="anesthesiologist_preference">
-					<label>Anesthesiologist Preference:</label>
-					<br>
-					<select name="pref_anest3" id="pref_anest3" class="PreferenceSelector">
-						<option selected="selected">No Preference</option>
-						@foreach($anesthesiologists as $a)
-							<option value="{{ $a->id }}">Dr. {{ $a->first_name }} {{ $a->last_name }}</option>
-						@endforeach
-					</select>
-				</div>
-				<br>
-	        @endif
+            @foreach ($datas as $data)
+                @if (is_null($data['schedule']))
+                    <h5>Preference #{{ $loop->iteration }}: None</h5>
```

Comment:
Forgot to mention that this used to say 1st, 2nd, 3rd, but will now say #1, #2, #3. I can't imagine this is a serious problem, but it could be viewed as a regression.